javascript-data-model/order-history.js

```diff
@@ -0,0 +1,93 @@
+var orderHistory = [
+  {
+    orderOneItems: [
```

Comment:
Instead of individually naming this for each object, I would just have a property named `items` on each object that stores the data for that order's items